Makefile

```diff
@@ -7,7 +7,8 @@ run:
 	# npx ts-node src/main.ts --base-dir=../large-monorepo --verbose
 
 lint:
-	npx eslint . --ext .ts
+	npx prettier --write .
+	npx eslint . --ext .ts --fix
```

Comment:
I was avoiding to do auto fix outside VSCode so the developer have more feeling of control over the code, so make lint was only for checking. But we can try the auto fix to see how it feels. What do you think?